packages/codemirror/src/editor.ts

```diff
@@ -540,6 +644,132 @@ export class CodeMirrorEditor implements CodeEditor.IEditor {
     this._editor.execCommand(command);
   }
 
+  /**
+   * Find definitions among tokens of this editor instance.
+   */
+  findDefinitions(name: string): Array<CodeEditor.IToken> {
```

Comment:
A lot of the logic in this function is specific to Python, which I don't think is really appropriate for JupyterLab core. We really want to take language-agnosticism as a central feature.